src/Components/RoutingMenu/RoutingMenu.jsx

```diff
@@ -610,194 +623,234 @@ function RoutingMenu({
   }
 
   return (
-    <div className="rd-routing-menu">
-      <Paper square elevation={3}>
-        <div style={{ height: 5 }}>
-          {showLoadingBar ? <LinearProgress /> : null}
-        </div>
-        <div className="rd-routing-menu-header">
-          <Tabs
-            value={DEFAULT_MOTS.includes(currentMot) ? currentMot : false}
-            className={classes.tabs}
-            onChange={(e, mot) => {
-              handleMotChange(e, mot, tracks);
-            }}
-            indicatorColor="primary"
-            textColor="primary"
-            aria-label="mots icons"
-          >
-            {currentMots.map(singleMot => {
-              return (
-                <Tab
-                  className={classes.tab}
-                  key={`mot-${singleMot.name}`}
-                  value={singleMot.name}
-                  icon={singleMot.icon}
-                  aria-label={singleMot.name}
-                  disabled={showLoadingBar}
-                />
-              );
-            })}
-          </Tabs>
-          {otherMots.length ? (
-            <FormControl className={classes.dropDown}>
-              <Select
-                renderValue={val => (val !== '' ? val : 'Other MOTs')}
-                className={classes.select}
-                classes={{ root: classes.selectInput }}
-                labelId="rd-other-mot-label"
-                value={currentOtherMot || ''}
-                disableUnderline={!currentOtherMot}
-                displayEmpty
-                onChange={changeCurrentOtherMot}
-                disabled={showLoadingBar}
+    <>
+      <div className="rd-routing-menu">
+        <Slide
+          direction={isDesktop ? 'right' : 'down'}
+          in={menuVisible}
+          onEntered={() => setIsEntered(true)}
```

Comment:
On Desktop view I think it's slightly ugly that the menu resizes after the Slide-In due to the button rendering afterwards. Ideal (IMO) would be if the button would slide in along with the menu. Maybe we can create a MenuToggleButton component and render it separately for open/closed menus?